x-pack/plugins/alerting/server/alerts_client/lib/build_new_alert.ts

```diff
@@ -65,13 +73,28 @@ export const buildNewAlert = <
             rule: rule.kibana?.alert.rule,
             status: 'active',
             uuid: legacyAlert.getUuid(),
+            workflow_status: get(cleanedPayload, ALERT_WORKFLOW_STATUS, 'open'),
             ...(legacyAlert.getState().duration
               ? { duration: { us: legacyAlert.getState().duration } }
               : {}),
-            ...(legacyAlert.getState().start ? { start: legacyAlert.getState().start } : {}),
+            ...(legacyAlert.getState().start
```

Comment:
hmmm ... I totally forgot about our issue with `start`, `end` and `duration` in `state`: https://github.com/elastic/kibana/issues/144929 ; the PR to fix this never got merged, I'd guess it's in a bad state by now.

Reply:
Yea, we never merged that PR :(